docs/guideline.md

```diff
@@ -0,0 +1,111 @@
+
+* [Explorer V2 Frontend](https://github.com/harmony-one/bounties/issues/37)
+* [Bridge dashboard with statistics](https://github.com/harmony-one/bounties/issues/26)
+* [ether.js as wrapper for RPC Provider](https://github.com/harmony-one/bounties/issues/20)
+* [Contract code verification tool](https://github.com/harmony-one/bounties/issues/27)
+
+##### Toolings
+
+* [Horizon Bridge Subgraphs](https://github.com/harmony-one/bounties/issues/15)
+* [Harmony Subgraphs: Transactions, HRC20 and HRC721](https://github.com/harmony-one/bounties/issues/10)
+* [analytics.harmony.one](https://github.com/harmony-one/bounties/issues/39)
+
+#### 1.2.2 External bounties
+
+##### External collaborations and integrations
+
+* Bridges ([Terra](https://github.com/harmony-one/bounties/issues/32), [Celo](https://github.com/harmony-one/bounties/issues/33))
+* API integration ([0x](https://github.com/harmony-one/bounties/issues/38), [Trezor](https://github.com/harmony-one/bounties/issues/29), [UMA](https://github.com/harmony-one/bounties/issues/12))
+* Misc ([Humming bot](https://github.com/harmony-one/bounties/issues/16))
+
+##### Ecosystem and DAPPS
+
+* [Argent Wallet](https://github.com/harmony-one/bounties/issues/31)
+* [Harmony Name Service](https://github.com/harmony-one/bounties/issues/13)
+* [PoolTogether](https://github.com/harmony-one/bounties/issues/44)
+* [Indonesia Rupiah Token (IDRT) stablecoin](https://github.com/harmony-one/bounties/issues/11)
+
+#### 1.2.3 Research bounties
+
+* [1s Finality](https://github.com/harmony-one/bounties/issues/14)
+* [nnet](https://github.com/harmony-one/bounties/issues/25)
+* [A lot more ideas](https://docs.google.com/spreadsheets/u/1/d/1J3gMrq52Ho75fpw716Am_5akjVsg6U5brHHSy1z4tDM/edit#gid=527101002)
+
+## 2 "Good" bounties
+
+### 2.1 What makes a good bounty?
+
+First, what is the characteristics of bounties?
+
+1. The candidates are all developers. And most of them work along.
+2. The candidate may not have prior knowledge to Harmony blockchain as well as the context.
+3. The candidate does not expect to run in long-term. They walk in and go.
+
+This will give several insights to what a good bounty should be:
+
+1. Self-contained.
+
+   This means the bounty would better not require help or support from anyone or anything else during the process, it is complete by itself. 
+
+   If the self-contained cannot be contained for a bounty, you need to provide with a clean scope with lesser boundary.
+
+   Sometimes this can be compromised through clean writing and some helpful conversation at the bootstrap. 
+
+2. No ambiguity in project description.
+
+   For engineers, less ambiguity the better. Treat them as high-level coding machine. You need to strictly define the starting point and target point, and let them improvise in the process. It would be very helpful to define the starting/ending with some engineering documentation.
+
+3. Well defined acceptance criteria.
```

Comment:
this is good.  for applications, i'd request a demo video.  for code change, we'd ask for unit test coverage and even e2e test scripts.